### PR TITLE
[draft-27] Django Admin Setup

### PR DESCRIPTION
**Pull request by aarajh**
**What?**

- Enabled django admin and setup database for development.
- Added support for our custom models in django admin.
- Updated main and /server readme files including better guide

**Why?**
- To be able to view and manage users from Django Admin GUI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
```
Comment (adil1508) [inline README.md]: ooh i like this new Readme

aaraj is v nyc <3

**Comment by adil1508** [comment]
Can we rebase off of master? 

I don't know why Github can't sigh. It doesn't look very conflicting to me :(

**Comment by aarajh** [comment]
> 
> 
> The only comment i have is whether we need the binary files under
> 
> Server/app/app/**pycache**/
> Server/app/core/**pycache**/
> Server/app/core/management/**pycache**/
> Server/app/core/management/commands/**pycache**/
> 
> or should these directories be in the .gitignore?

True we dont. Fixed now. Added python .gitignore and removed the unnecessary files

File: Server/app/core/admin.py [hunk 2]
```diff
@@ -1,3 +1,33 @@
-# from django.contrib import admin
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+from core import models
+from django.utils.translation import gettext as _
 
-# Register your models here.
```
Comment (adil1508) [inline Server/app/core/admin.py]: Why u hate this line :'(

Comment (aarajh) [reply Server/app/core/admin.py]: Because it was in the auto-template :p

**Comment by adil1508** [comment]
lgtm 🚀

**Pull request merged**